### ISSUE TITLE
Report warnings if the weren't fixed by autofix in `order`

### ISSUE THREAD

**Issue by hudochenkov** [hudochenkov/stylelint-order#85]
Similar to the [`properties-order`](https://github.com/hudochenkov/stylelint-order/commit/5c03c7318cf1581cc17f457d7b96f223e84fe04e) `order` should report warning which weren't fixed by autofix. It could happen in CSS-in-JS, if rule has interpolations. Then `postcss-sorting` will skip it to not break the users code.

```js
{ order: ['declarations', 'rules'] }
```

```js
const Component = styled.div`
	a {
		color: blue;
	}
	${interpolation}
	color: tomato;
`;
```
